app/services/trainees/create_from_apply.rb

```diff
@@ -144,6 +146,19 @@ def nationality_names
       end
     end
 
+    def start_academic_cycle(course)
```

Comment:
This is bleeding what start and end academic cycle means into this class (and the the HESA version) which we don't want to do. We already have the `Trainees::SetAcademicCycles` service that handles all of that. I think we just need to call that service as a step in the call method of this class (around [here](https://github.com/DFE-Digital/register-trainee-teachers/pull/2517/files#diff-990dcf7855f4e8e18c122b1f32795904d62637685a5059fc0eb557a5f9d24f25L29-L32) somewhere) and then do a similar thing in the HESA version.